clusters/app.ci/quayio-pull-through-cache/quayio-pull-through-cache-us-east-1.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+  spec:
+    to:
+      kind: Service
+      name: quayio-pull-through-cache-us-east-1
+    tls:
+      termination: Edge
+      insecureEdgeTerminationPolicy: Redirect
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+  spec:
+    selector:
+      app: quayio-pull-through-cache-us-east-1
+    ports:
+    - port: 5000
+      targetPort: 5000
+    type: ClusterIP
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: quayio-pull-through-cache-us-east-1
+    namespace: ci
+    labels:
+      app: quayio-pull-through-cache-us-east-1
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: quayio-pull-through-cache-us-east-1
+    template:
+      metadata:
+        labels:
+          app: quayio-pull-through-cache-us-east-1
+      spec:
+        volumes:
+        - name: config
+          configMap:
+            name: quayio-pull-through-cache-config
+        - name: exec
+          configMap:
+            name: quayio-pull-through-cache-exec
+        containers:
+        - name: quayio-cache
+          image: docker.io/registry:2.8.1
```

Comment:
this is to be changed

Reply:
+1 . I'd definitely want to burn `apache2-utils` into the image . If this pod can't start, the impact to CI is enormous. Imagine the package name changing or a rate limiting affecting the install -- preventing the pod from coming up.